app/models/optimized_image.rb

```
@@ -139,25 +139,24 @@ def self.downsize_instructions_animated(from, to, dimensions, opts={})
-    optimize("resize", from, to, width, height, opts)
+    optimize("resize", from, to, "#{width}x#{height}", opts)
   end
 
   def self.downsize(from, to, max_width, max_height, opts={})
-    optimize("downsize", from, to, max_width, max_height, opts)
+    optimize("downsize", from, to, "#{max_width}x#{max_height}", opts)
   end
 
-  def self.optimize(operation, from, to, width, height, opts={})
-    dim = dimensions(width, height)
+  def self.downsize(from, to, dimensions, opts={})
```

RuboCop (1.76.1)

[warning] Method `OptimizedImage.downsize` is defined at both app/models/optimized_image.rb:145 and app/models/optimized_image.rb:149.

(Lint/DuplicateMethods)